src/morphology.cpp

```diff
@@ -34,7 +34,6 @@ Morphology::Morphology(const Property::Properties& properties, unsigned int opti
     // their respective loaders
     if (properties._cellLevel.fileFormat() == "h5") {
         mut::Morphology mutable_morph(*this);
-        mutable_morph.sanitize();
```

Comment:
hopefully you did it this way via a sanitize function...

Reply:
Yep. It's already exposed but with the latest discussion I don't know if it should remain so or if we should make it private again ?

Reply:
I don t think so. This will probably be used to "sanitize" the morph for the "safe zone".
In my mind we will have people doing : 
```
morph.sanitize().write()
```